packages/starknet-snap/src/estimateFee.ts

```diff
@@ -108,35 +107,21 @@ export async function estimateFee(params: ApiParamsWithKeyDeriver) {
       ];
     }
 
-    let estimateFeeResp;
+    const estimateBulkFeeResp = await estimateFeeBulk(
+      network,
+      senderAddress,
+      senderPrivateKey,
+      bulkTransactions,
+    );
+    logger.log(
+      `estimateFee:\nestimateFeeBulk estimateBulkFeeResp: ${toJson(
+        estimateBulkFeeResp,
+      )}`,
+    );
+    const estimateFeeResp = addFeesFromAllTransactions(estimateBulkFeeResp);
 
-    if (accountDeployed) {
-      // This condition branch will be removed later when starknet.js
-      // supports estimateFeeBulk in rpc mode
-      estimateFeeResp = await estimateFeeUtil(
-        network,
-        senderAddress,
-        senderPrivateKey,
-        txnInvocation,
-      );
-      logger.log(
-        `estimateFee:\nestimateFeeUtil estimateFeeResp: ${toJson(
-          estimateFeeResp,
-        )}`,
-      );
-    } else {
-      const estimateBulkFeeResp = await estimateFeeBulk(
-        network,
-        senderAddress,
-        senderPrivateKey,
-        bulkTransactions,
-      );
-      logger.log(
-        `estimateFee:\nestimateFeeBulk estimateBulkFeeResp: ${toJson(
-          estimateBulkFeeResp,
-        )}`,
-      );
-      estimateFeeResp = addFeesFromAllTransactions(estimateBulkFeeResp);
+    if (!estimateFeeResp.suggestedMaxFee || !estimateFeeResp.overall_fee) {
```

Comment:
it looks like a never case , as either it fail in starknet, but it wont return nth